pkgs/development/libraries/gtk/4.x.nix

```diff
@@ -131,8 +131,9 @@ stdenv.mkDerivation (finalAttrs: {
     libXi
     libXrandr
     libXrender
-  ]) ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    AppKit
+  ]) ++ lib.optionals (stdenv.hostPlatform.isDarwin
+                    && lib.versionOlder apple-sdk.version "10.15") [
+    apple-sdk_10_15
```

Comment:
It looks like they don’t have any version conditional to only use this picker on 10.15, so I think we should probably also use `(darwinMinVersionHook "15.0")`, and add both to `propagatedBuildInputs` so that downstream derivations get the appropriate SDK and deployment target.

Reply:
yeah, i noticed the above typo and made a note to use `10.15` and thought i double checked and still managed to do it wrong...